### ISSUE TITLE
Better visual hierarchy/UI for Subscription page

### ISSUE THREAD

**Issue by pftchal** [universelabs/universe.engineering#61]
**Description**
1. Subscription page: The intro of Universe could be moved closer to the logo. And try to differentiate that from the intro of newsletters by changing the font size/color. 

2. It is a bit wordy for the "Email Confirmation Page", try to shorten it like: " To confirm your subscription, click the link in the email we just sent."

3. "Done" button should be linked to the app website in the future.

4. The content in the email could be more specific: 

"Please confirm subscription to mailing list for upcoming releases, news &more from Universe."

Because newsletters in general, sometimes can be annoying, it is important to keep users motivated throughout the process to subscribe, they might lose interest at any point so I think it will be nice to remind them the perks of subscribing. 

**Screenshots**
![Screen Shot 2019-08-06 at 11 28 06 AM](https://user-images.githubusercontent.com/48525282/62553748-e837d600-b83d-11e9-8705-76ec891e1f8f.png)
![Screen Shot 2019-08-06 at 11 30 15 AM](https://user-images.githubusercontent.com/48525282/62553753-e9690300-b83d-11e9-97fc-d6b6b466556e.png)

**Comment by pftchal** [comment]
@guylepage3

**Comment by guylepage3** [comment]
Thanks @pftchal.. Good call. 👍
